content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
+
+This mechanism is called the _Unknown Version Interoperability Proxy_.
```

Comment:
Remove? This line feels redundant.

Reply:
Done

Reply:
Hi. Docs tech lead here.
Please put that text back. It will help readers to understand how that text links to the page, and wraps up the intro.

@Richabanker sorry for the mixed messaging on this.

Reply:
No worries, updated.